src/component/axis/parallelAxisAction.ts

```diff
@@ -25,14 +25,20 @@ import ParallelModel from '../../coord/parallel/ParallelModel';
 import { EChartsExtensionInstallRegisters } from '../../extension';
 
 interface ParallelAxisAreaSelectPayload extends Payload {
-    parallelAxisId: string;
-    intervals: ParallelAxisInterval[]
+    parallelAxisId?: string;
+    intervals?: ParallelAxisInterval[];
+    axes?: string[]
 }
 
 const actionInfo = {
     type: 'axisAreaSelect',
-    event: 'axisAreaSelected'
-    // update: 'updateVisual'
+    event: 'axisAreaSelected',
+    update: 'updateVisual'
```

Comment:
Can you explain why `update: 'updateVisual'` should be uncommented? I don't know why it's originally commented.